style.css

```
@@ -0,0 +1,143 @@
+@font-face {
+  font-family: "Pretendard-Regular";
+  src: url("https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff")
+    format("woff");
+  font-weight: 400;
+  font-style: normal;
+}
+* {
+  font-family: "Pretendard-Regular";
+}
+button {
+  background-color: #3692ff;
+}
+button:hover {
+  cursor: pointer;
+  background-color: #1967d6;
+}
+button:active {
+  background-color: #1251aa;
+}
+header {
+  padding: 0.8em;
+  margin: auto;
+}
+nav {
+  width: calc(100% - 200px);
+  height: 2.5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+}
+nav img {
+  width: 10em;
+}
+
+nav > button {
+  padding: 0.8em 2em;
+  border-radius: 8px;
+  background: #3692ff;
+  font-size: 1em;
+}
+nav > button > a {
+  color: white;
+}
+.main-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.banner {
+  background-color: #cfe5ff;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  padding: 10em 0 0;
+}
+
+.banner h1 {
+  letter-spacing: -0.1em;
+  font-size: 2.5em;
+}
+.banner img {
+  width: 50%;
+}
+#banner-content {
+  position: relative;
+  bottom: 0;
+}
+.banner h1 {
+  font-size: 2.5em;
+  font-weight: 700;
+  line-height: 1.5;
+}
+.banner button {
+  padding: 0.8em 6em;
+  border-radius: 999px;
+}
+.banner > div > button > a {
+  color: white;
+  font-weight: 300;
+}
+.card-box {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 10em 0 15em;
+}
+.card {
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fcfcfc;
+  flex-grow: 1;
+}
+
+.card-title {
+  font-size: 1.1em;
+  color: #3692ff;
+  font-weight: 700;
+}
+
+.card h2 {
+  margin: 0.2em 0;
+  line-height: 1.4;
+  text-align: left;
+  font-size: 2.5em;
+  font-weight: 700;
+}
+.card-box:nth-child(2),
+.card-box:nth-child(2) h2 {
+  text-align: right;
+}
+
+.card p {
+  color: #374151;
+  line-height: 1.4;
+}
+.card > img {
+  width: 50%;
+}
+footer {
+  background-color: #111827;
+  width: 100%;
+  height: 11.875em;
+  display: flex;
+  justify-content: space-around;
+  padding: 1.5em;
```

❗️ 수정요청
해당 태그가 width를 100%로 가지고 있는데 여기에 padding이 더해지니 overflow가 생기게 됩니다.
배포 사이트에서 보시면 가로 스크롤이 생긴 것을 확인할 수 있어요~
이를 해결할 수 있는 방법들은 여러가지가 있는데 현재 html 구조를 유지하면서 가장 쉬운 방법은 `box-sizing: border-box`를 추가하시는 겁니다~

```suggestion
  padding: 1.5em;
  box-sizing: border-box;
```